pkg/orchestration/wiring/k8scompute/plugin.go

```diff
@@ -596,9 +597,10 @@ func buildAntiAffinity(labelMap map[string]string) *core_v1.PodAntiAffinity {
 	}
 }
 
-func buildPodDisruptionBudgetSpec(labelMap map[string]string, minReplicas int32) policy_v1.PodDisruptionBudgetSpec {
+func buildPodDisruptionBudgetSpec(labelMap map[string]string, minReplicas int32) (policy_v1.PodDisruptionBudgetSpec, string) {
 	// Make sure we can drain nodes if the minReplicas is < 3
 	var minAvailable string
+	var postfix string
```

Comment:
nit: no need to declare postfix here, can be declared below when you set the value

Reply:
Fixed.